organizations.json

```diff
@@ -7,6 +7,11 @@
   {
     "name": "Renzo",
     "identifier": "urn:oid:2.16.840.1.113883.2.4.6.1:13579246",
-    "publicKey": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApXmCBIrbPKY4sPin5m1jUFy9Ht9eil9kvxoUkMspQbKMTzAASgUbXQflDLzc1aJ5VpugaS3c0XVhkpxVtxH5Sd0Q1VFeIB28VmpCbcm8Gh7TwPwsZ0JBY4CL6P1dnJyO7ZRkOzppj03UMjGzqff6tiK6FU1Dcw5QK2XZiJJKmndOPvuwC0RoeaR1IHSU8Q4iOxiaR3OZBtcSw2oiLMxCwhgGhzfYu+geqZ0VmdCHG5w30f99S8i4qvJb16XRBy2g2p69NbWMbFOT4LxcRnxXAxH6VTiMRWra49W9anYLhT+2FWiJS6cu0V8A1Ro0MQV3nzUhfpZEnW+lKcO/Fp8RJQIDAQAB\n-----END PUBLIC KEY-----",
+    "publicKey": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApXmCBIrbPKY4sPin5m1jUFy9Ht9eil9kvxoUkMspQbKMTzAASgUbXQflDLzc1aJ5VpugaS3c0XVhkpxVtxH5Sd0Q1VFeIB28VmpCbcm8Gh7TwPwsZ0JBY4CL6P1dnJyO7ZRkOzppj03UMjGzqff6tiK6FU1Dcw5QK2XZiJJKmndOPvuwC0RoeaR1IHSU8Q4iOxiaR3OZBtcSw2oiLMxCwhgGhzfYu+geqZ0VmdCHG5w30f99S8i4qvJb16XRBy2g2p69NbWMbFOT4LxcRnxXAxH6VTiMRWra49W9anYLhT+2FWiJS6cu0V8A1Ro0MQV3nzUhfpZEnW+lKcO/Fp8RJQIDAQAB\n-----END PUBLIC KEY-----"
+  },
+  {
+    "name": "Ton hoekje",
+    "identifier": "urn:oid:2.16.840.1.113883.2.4.6.1:123123123",
+    "publicKey": "-----BEGIN PUBLIC KEY-----MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvLq3nkxQSHdiZ6ffrQh7dz4WNaSs2ou1en2LF3vTREulinWCMx1H8aodYxvLmzp8vDx7/Mum7/11aYoNko8ZxhG2fMxMJr9ANCQCo8OCMV6Lumny+smJh17K1NOq6KRjkpJ2LcMmoORSxQjFaWQ7uj/z8TTC7/d9GmhD5XuhGiHRbQ7jnDLuUFb53hbm3oPyCxCGHTBg5ujepS2WbHnFKVdzimuT+iXoWi91Ueze6Zf/6IyslVywLzFzalUwx3EOtA35PrtSVGuWZnZMepIdBq8zRAqF3xDJo5dUc8rrg4wvyRMdNPDtXSpjOHJoBT1LRat+wZE28O8RZUxIjqmXZQIDAQAB-----END PUBLIC KEY-----"
```

Comment:
public keys should contain newlines \n chars on each newline.

Reply:
\n 's toegevoegd